src/dc_msg.rs

```diff
@@ -177,15 +177,15 @@ pub unsafe fn dc_get_msg_info(context: &Context, msg_id: u32) -> *mut libc::c_ch
     }
     free(p as *mut libc::c_void);
 
-    if (*msg).type_0 != DC_MSG_TEXT {
+    if (*msg).type_0 != Viewtype::Text {
         ret += "Type: ";
         match (*msg).type_0 {
-            DC_MSG_AUDIO => ret += "Audio",
-            DC_MSG_FILE => ret += "File",
-            DC_MSG_GIF => ret += "GIF",
-            DC_MSG_IMAGE => ret += "Image",
-            DC_MSG_VIDEO => ret += "Video",
-            DC_MSG_VOICE => ret += "Voice",
+            Viewtype::Audio => ret += "Audio",
```

Comment:
you can use https://github.com/Peternator7/strum#display to derive a to_string method, avoiding the manual match

Reply:
Fine. Note, debug messages will change format.